cmd/juju/storage/remove.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package storage
+
+import (
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/juju/common"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// NewRemoveStorageCommandWithAPI returns a command
+// used to remove storage from the model.
+func NewRemoveStorageCommandWithAPI() cmd.Command {
```

Comment:
You're referring to code employing bad practices, using export_test to override hard-coded behaviour. We should
- not use export_test.go
- expose the dependencies of the command (i.e. as in NewRemoveStorageCommand)

The NewRemoveStorageCommandWithAPI command probably shouldn't even exist. It's there because of how the commands are structured, with the API connection logic embedded.